app/connectors/SubmissionConnector.scala

```diff
@@ -26,18 +27,30 @@ import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
 
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
-import scala.xml.transform.{RewriteRule, RuleTransformer}
 import scala.xml.{Elem, Node, NodeSeq}
+import scala.xml.transform.{RewriteRule, RuleTransformer}
 
 class SubmissionConnector @Inject() (httpClient: HttpClient, config: FrontendAppConfig) extends Logging {
 
-  val submitUrl = s"${config.mdrUrl}/mandatory-disclosure-rules/submit"
+  val submitUrl    = s"${config.mdrUrl}/mandatory-disclosure-rules/submit"
+  val submitxmlUrl = s"${config.mdrUrl}/mandatory-disclosure-rules/submitxml"
+
+  def submitDocument(submissionDetails: SubmissionDetails)(implicit
+    hc: HeaderCarrier,
+    ec: ExecutionContext
+  ): Future[Option[ConversationId]] =
+    httpClient.POST[SubmissionDetails, HttpResponse](submitUrl, submissionDetails) map {
+      case response if is2xx(response.status) => Some(response.json.as[ConversationId])
+      case errorResponse =>
+        logger.warn(s"Failed to submitDocument: revived the status: ${errorResponse.status} and message: ${errorResponse.body}")
+        None
+    }
 
-  def submitDocument(fileName: String, enrolmentID: String, xmlDocument: NodeSeq, fileSize: Option[Long] = None)(implicit
+  def submitxmlDocument(fileName: String, enrolmentID: String, xmlDocument: NodeSeq, fileSize: Option[Long] = None)(implicit
```

Comment:
Can this method be moved to a new connector in the existing testOnlyDoNotUseInAppConf folder, since that's the only place this should be used with the postman script.